all_msgs/ros2interop/ros-types-all.idl

```diff
@@ -0,0 +1,1825 @@
+module std_msgs {
```

Comment:
We usually use PascalCase for IDL file names. Are file names something that is mandated or something we get to choose?

Reply:
Also, I like the name `all_msgs`, but I thin the `ros2interop` folder is unneeded. 

If this only includes msg types then I would rename `ros2interop` to `msg`.

Reply:
Good points.   This needs another look about how to clarify which parts are ROS compatible, and which are ROS 2 compatible.
I'll look into this more & will modify the pull request.